Makefile

```diff
@@ -18,3 +18,16 @@ install-submodules:
 update-submodules:
 	@printf $(COLOR) "Updatinging temporal and tctl submodules..."
 	git submodule update --force --remote $(PROTO_ROOT)
+	
```

Comment:
What's about `buildx` targets? Please just move them all. They are there for reason. Not for automation, but for the case if one wants to build image locally.

Reply:
added